lib/ahoy/ahoy.js

```diff
@@ -82,8 +82,8 @@ export default class Ahoy {
   }
 
   initConnection = async () => {
-    this.hasInternetAccess = await NetInfo.isConnected.fetch();
-    NetInfo.isConnected.addEventListener('connectionChange', this.handleNetInfoConnectionChange);
```

Comment:
I do not understand why deleted connection listener

Reply:
hasInternetAccess is used only in 3 places: `addTrackingWorker`, `addVisitorWorker`, `createJob`.

`addTrackingWorker`, `addVisitorWorker` are called only once at start, so we don't need to watch for the internet connection for them.

`createJob` has this code:
`createJob = (name, event) => {
    if (this.offlineMode) this.hasInternetAccess = undefined;
    if (this.queue) this.queue.createJob(name, event);
  };`

And `hasInternetAccess` is useless here